ajax/libs/material-datetime-picker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "material-datetime-picker",
+  "filename": "material-datetime-picker.min.js",
+  "description": "A Vanilla JS Material Design date/time picker component",
+  "homepage": "https://ripjar.github.io/material-datetime-picker/",
+  "keywords": [
+    "material-design",
+    "date-picker",
+    "datetime-picker",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ripjar/material-datetime-picker"
+  },
+  "license": "MIT",
+  "author": "Joe Whitfield-Seed <joeseed86@gmail.com>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ripjar/material-datetime-picker.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*picker!(.mjs)*"
+        ]
+      }
+    ]
+  }
```

Comment:
We prefer using npm auto-update over git auto-update. Also, in this case, [npm](https://www.npmjs.com/package/material-datetime-picker) has one more release than GitHub.

```
"npmName": "material-datetime-picker",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*picker!(.mjs)*"
      ]
    }
  ]
```

You can also refer to the [auto-update documentation](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md) for more info.